superset/models/helpers.py

```diff
@@ -1331,7 +1334,8 @@ def get_sqla_query(  # pylint: disable=too-many-arguments,too-many-locals,too-ma
 
         columns_by_name: Dict[str, "TableColumn"] = {
             col.get("column_name"): col
-            for col in self.columns  # col.column_name: col for col in self.columns
+            # col.column_name: col for col in self.columns
+            for col in self.columns
```

Comment:
shouldn't this be `for col in self.columns:` it's missing the colon

Reply:
It looks like it's the end of a dictionary comprehension, not a for loop, so no colon needed. The comment change btw was just part of my automated linting on line length. I'm not actually sure what the comment means, too. :)